scripts/docker-compose/imports/smartdocuments-wiremock/__files/sdapi-structure-response-body.json

```diff
@@ -0,0 +1,227 @@
+                }
+              ],
+              "accessible": true,
+              "id": "348097107FA346DC9AEBBE33A5500B79",
+              "name": "OpenZaak"
+            },
+            {
+              "templateGroups": [],
+              "templates": [
+                {
+                  "favorite": false,
+                  "id": "7B7857BB9959470C82974037304E433D",
+                  "name": "Data Test"
+                },
+                {
+                  "favorite": false,
+                  "id": "273C2707E5A844699B653C87ACFD618E",
+                  "name": "OpenZaakTest"
+                }
+              ],
+              "accessible": true,
+              "id": "CC2696396E1D4899A74AC025B19C8FDC",
+              "name": "Indienen aansprakelijkstelling door derden behandelen"
+            },
+            {
+              "templateGroups": [],
+              "templates": [
+                {
+                  "favorite": false,
+                  "id": "7B7857BB9959470C82974037304E433D",
+                  "name": "Data Test"
+                },
+                {
+                  "favorite": false,
+                  "id": "273C2707E5A844699B653C87ACFD618E",
+                  "name": "OpenZaakTest"
+                }
+              ],
+              "accessible": true,
+              "id": "DA3A76D24DFD48C9837B03E47BC701FB",
+              "name": "Melding evenement organiseren behandelen"
+            }
+          ],
+          "templates": [
+            {
+              "favorite": false,
+              "id": "2F426D9B06034FDB93FDC2C6427640DD",
+              "name": "Aanvullende informatie nieuw"
+            },
+            {
+              "favorite": false,
+              "id": "CDC0FD0F08B545B59E36D610AFFA0B58",
+              "name": "Aanvullende informatie oud"
+            },
+            {
+              "favorite": false,
+              "id": "40A8482A34F941618B95359BF8246FB8",
+              "name": "Advies nieuw"
+            },
+            {
+              "favorite": false,
+              "id": "16B8B5A8E18A4BA484302029B51C32B6",
+              "name": "Advies oud"
+            },
+            {
+              "favorite": false,
+              "id": "53AFB00EE74E4B8AB1EC73CCB3251F8B",
+              "name": "Besluit nieuw"
+            },
+            {
+              "favorite": false,
+              "id": "60D711D768184F7F94A8BC1553A689A1",
+              "name": "Besluit oud"
+            },
+            {
+              "favorite": false,
+              "id": "7B7857BB9959470C82974037304E433D",
+              "name": "Data Test"
+            },
+            {
+              "favorite": false,
+              "id": "6BF6F7B751554C24964C7B39DC62F2BA",
+              "name": "Memo nieuw"
+            },
+            {
+              "favorite": false,
+              "id": "4A615E0E9F754539A7BB969B55CC814A",
+              "name": "Memo oud"
+            },
+            {
+              "favorite": false,
```

Comment:
Maybe ok but do we really need all these templates here for our itests? Usually when you have 2 or some data item you are testing enough and it does not make sense to add any more? I am just a bit worried about the maintenance of test data like this so would rather keep it as small as we can.

Reply:
Left only 2 templates